src/lib/connectors/magicConnectConnector.ts

```diff
@@ -40,10 +45,11 @@ export class MagicConnectConnector extends Connector {
 
   // Private method to initialize the Magic instance
   private initializeMagicInstance() {
-    const { apiKey, magicSdkConfiguration } = this.options
+    const { apiKey, magicSdkConfiguration, networks } = this.options
     if (typeof window !== 'undefined') {
       this.magic = new Magic(apiKey, {
         ...magicSdkConfiguration,
+        network: magicSdkConfiguration?.network || networks?.[0],
```

Comment:
Add a fallback to the list of networks so it is now possible only to define the list of networks